views/editor/bramble.html

```diff
@@ -36,6 +36,10 @@
             animation: loadingpulse 1s infinite linear;
             -webkit-animation: loadingpulse 1s infinite linear;
         }
+	#spinner-container .loading-message-help-link {
```

Comment:
Please use spaces instead of tabs here and match the indentation in the rest of the file, this block appears like this in my editor...

![image](https://cloud.githubusercontent.com/assets/25212/23238545/48b88850-f917-11e6-9b38-db18c4f4d2a9.png)